tests/bb_bf_onehot_post.py

```diff
@@ -0,0 +1,16 @@
+print("VOLTAGE    CPU(re,im)       CPU(re,im)sh     CPU       GPU")
```

Comment:
It would be good to have some comments on what this script does, and where it gets used.

Reply:
done; it aint pretty